### PR TITLE
[Agent] Using raw pointer in Packet

### PR DESCRIPTION
**Pull request by yuanchaoa**
### This PR is for:

- Agent

#### Checklist
- [ ] Added unit test.
#### Backport to branches
- main

### CODE REVIEW

File: agent/src/dispatcher/mirror_mode_dispatcher.rs [hunk 1]
```diff
@@ -412,6 +413,7 @@ impl MirrorModeDispatcher {
                 if self.base.tap_interface_whitelist.next_sync(Duration::ZERO) {
                     self.base.need_update_bpf.store(true, Ordering::Relaxed);
                 }
+                drop(recved);
```
Comment (rvql) [inline agent/src/dispatcher/mirror_mode_dispatcher.rs:416]: 几个`drop`不需要写，离开作用域自动调

Comment (yuanchaoa) [reply agent/src/dispatcher/mirror_mode_dispatcher.rs:416]: 不写不行，会报错
image

Comment (yuanchaoa) [reply agent/src/dispatcher/mirror_mode_dispatcher.rs:416]: image

File: agent/crates/public/src/packet.rs [hunk 2]
```diff
@@ -20,13 +20,26 @@ use std::time::Duration;
...
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
 pub struct Packet<'a> {
     pub timestamp: Duration,
     pub if_index: isize,
     pub capture_length: isize,
     pub data: &'a mut [u8],
-    pub raw: Option<Vec<u8>>,
+    pub raw: Option<*mut u8>,
```
Comment (rvql) [inline agent/crates/public/src/packet.rs:29]: 加一些注释描述`data`和`raw`的关系

**Review by rvql — APPROVED** [review]

**Pull request merged**